inspec/a2-iam-v2-integration/controls/iam_v2.rb

```diff
@@ -1036,9 +1092,6 @@
       }
 
       before(:all) do
-        puts "wat"
```

Comment:
😂 the desperate cry of someone trying to debug these inspec tests..

Reply:
I think that was me too :sob: